### ISSUE TITLE
Creating a papertrail service fails with error 500

### ISSUE THREAD

**Issue by kjw** [convox/rack#392]
Trying this:

    edgar:~ karl$ convox services create papertrail --name pt1 --url <url>

I get:

    Creating pt1 (papertrail: name="pt1" url="<url>")... ERROR: response status: 500

Version info:

    edgar:~ karl$ convox -v
    client: 20160226022548
    server: 20160225233627 (grid.convox.com)

    edgar:~ karl$ convox rack
    Name     labs
    Status   running
    Version  20160225233627
    Count    3
    Type     t2.small

**Comment by kjw** [comment]
I also get a 500 error when trying to add the Slack integration via grid. 

I'm running the rack in `eu-west-1`.

**Comment by nzoschke** [comment]
Looks like a bug with the new StackName prefix stuff. I don't see a good workaround short of fixing the root cause.

**Comment by nzoschke** [comment]
I was planning on doing lots of service work this week, but can't get to this today yet.

**Comment by xtfxme** [comment]
I'm seeing this as well on our racks and will take a look.

**Comment by xtfxme** [comment]
@nzoschke

I didn't open a PR because it's trivial and I figure you might want to add tests. I specifically remember fixing this logic error when I reworked the `controllers.AppCreate` but it must have persisted here. If you look at the impl there you'll see the proper check for `err == nil`.

```diff
diff --git a/api/controllers/services.go b/api/controllers/services.go
index 4cc93d9..ddbbdc3 100644
--- a/api/controllers/services.go
+++ b/api/controllers/services.go
@@ -58,7 +58,7 @@ func ServiceCreate(rw http.ResponseWriter, r *http.Request) *httperr.Error {
        // Early check for unbound service only.
        service, err := models.GetServiceUnbound(name)
 
-       if err != nil {
+       if err == nil {
                return httperr.Errorf(403, "there is already a legacy service named %s (%s). We recommend you delete this service and create it again.", name, service.Status)
        }
 
```

**Comment by xtfxme** [comment]
I have applied the above fix to our local racks and can confirm things now work as expected.

**Comment by nzoschke** [comment]
The fix for this is here: https://github.com/convox/rack/pull/406

I'll close this issue when that is merged and released.

**Comment by nzoschke** [comment]
This is fixed and released in 20160302193502

**Comment by kjw** [comment]
Thank you and keep up the good work.
